source/PlayerInfo.cpp

```diff
@@ -2112,6 +2112,28 @@ void PlayerInfo::SelectNext()
 
 
 
+void PlayerInfo::DeselectAllSecondaries()
+{
+	selectedWeapons.clear();
+}
+
+
+
+void PlayerInfo::ToggleAnySecondary(const Outfit *outfit)
+{
+	if(!flagship || flagship->Outfits().empty())
```

Comment:
What's the reason why we are checking this?

Reply:
It's done in SelectNext() (renamed to SelectNextSecondary() in this PR). I'm assuming it's there so you can't cycle the weapons when your flagship is destroyed, the second part might not be relevant here.
While we're here, this method doesn't a tually make sure that the ship has the outfit the method is passed, is htat an issue?

Reply:
I don't think it's an issue no. But yeah the second condition looks unnecessary indeed.

Reply:
Removed